src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,9 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = new UserService().findByEmail(email);
```

Comment:
```suggestion
        User user = userService.findByEmail(email);
```

when you use `new UserService()` - you create new object, you already have it (class-level variable)